test-data/basic.rpl

```diff
@@ -131,7 +131,7 @@ RANGE_BEGIN 0 100
 ENTRY_BEGIN
 MATCH opcode qtype qname
 ADJUST copy_id copy_query
-REPLY QR RD NOERROR
+REPLY QR RD RA NOERROR
```

Comment:
Was the original Unbound .rpl file wrong here?

Reply:
This was already a custom test and not meant to be an existing test

Reply:
Ah, my bad.